IdentityCore/src/webview/operations/MSIDWebResponseBrokerInstallOperation.m

```diff
@@ -0,0 +1,102 @@
+// Copyright (c) Microsoft Corporation.
+// All rights reserved.
+//
+// This code is licensed under the MIT License.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files(the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and / or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions :
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+
+#import "MSIDWebResponseBrokerInstallOperation.h"
+#import "MSIDWebviewResponse.h"
+#import "MSIDBrokerInteractiveController.h"
+#import "MSIDInteractiveTokenRequestParameters.h"
+#import "MSIDWebWPJResponse.h"
+#import "MSIDTokenRequestProviding.h"
+#import "MSIDKeychainTokenCache.h"
+#import "MSIDLegacyTokenCacheAccessor.h"
+#import "MSIDDefaultTokenCacheAccessor.h"
+#import "MSIDAADV2Oauth2Factory.h"
+#import "MSIDDefaultTokenResponseValidator.h"
+#import "MSIDAccountMetadataCacheAccessor.h"
+
+#if TARGET_OS_IPHONE
```

Comment:
if this entire class is only needed on iOS, it should just live in iOS folder. 
Separate question is to confirm that it's indeed not needed on macOS.
I believe previous code was executing on macOS as well, but I might be mistaken.

Reply:
This class is used on both Mac and iOS and the above ```#if TARGET_OS_IPHONE``` should be removed as it is legacy from previous code changes.

However, we check if runs on mac, the code will send back errors in ```invokeWithInteractiveTokenRequestParameters``` which is aligned with the original implementations

Reply:
Removed